extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/DevServicesConfig.java

```diff
@@ -22,6 +22,12 @@ public class DevServicesConfig {
     @ConfigItem(defaultValue = "true")
     public boolean enabled = true;
 
+    /**
+     * Use lightweight dev services instead of Keycloak
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean lightweight = true;
```

Comment:
`lightweight = true` while `defaultValue = "false"`, can you elaborate why one is `true` and the other one is `false`? FWIW Keycloak startup on my workstation is quite slow, I'd also set it to true by default, but I read in previous comments Sergey reservations.

Reply:
You're right, this was confusing, the default value should be `false` unless we can convince people to turn it to `true.` Perhaps just start like this and enable it in the future after people try it out?

Or perhaps we make it default to true unless people have keycloak dev service configuration that the lightweight version does not support?

Reply:
@FroMage Lets keep it disabled by default for now, until we get the feedback and tune minor follow issues. Over time we can enable by default if the lightweight option starts dominating